pkg/server/server.go

```diff
@@ -150,16 +154,80 @@ func (s *Server) ListenAndServe(ctx context.Context, logger log.Logger, port str
 		return fmt.Errorf("failed to register pprof handlers: %w", err)
 	}
 
+	// Strip the subpath
 	uiFS, err := fs.Sub(ui.FS, "packages/app/web/dist")
 	if err != nil {
 		return fmt.Errorf("failed to initialize UI filesystem: %w", err)
 	}
 
+	// There is currently no way to go between `http.FileServer(http.FS(uiFS))` and execute
+	// templates an http.ServeMux with registered paths and executes templates is used.
+	uiHandler := http.ServeMux{}
```

Comment:
I think we should create a new method to init `uiHandler` and extract the logic there.

Reply:
:+1: changed